src/main/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceService.scala

```diff
@@ -92,22 +95,22 @@ trait WorkspaceService extends HttpService with FireCloudDirectives {
     value = "import entities (JSON)",
     nickname = "importEntitiesJSON",
     httpMethod = "POST",
-    response = classOf[EntityCreateResult],
+    response = classOf[Seq[EntityCreateResult]],
```

Comment:
You could keep the  response = classOf[EntityCreateResult] and add another line for responseContainer = "Seq" instead of merging them. 

On a somewhat related note, we have not been adding these to our orchestration services due to the high churn on the model classes. Also, given that there is a lot of code duplication since we have to re-write the case class in orchestration to match what's in other service endpoints - we have been avoiding that and waiting for some kind of dsde-commons code to come about so we could reference the model classes there. Feel free to approach this as you see fit - we don't have much insight into the level of model churn in this case.